digdag-standards/src/test/java/io/digdag/standards/scheduler/DailySchedulerTest.java

```diff
@@ -0,0 +1,222 @@
+package io.digdag.standards.scheduler;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import io.digdag.client.config.Config;
+import io.digdag.client.config.ConfigFactory;
+import io.digdag.spi.Scheduler;
+import io.digdag.spi.ScheduleTime;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import static java.util.Locale.ENGLISH;
+import static io.digdag.client.DigdagClient.objectMapper;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class DailySchedulerTest
+{
+    static Config newConfig()
+    {
+        return new ConfigFactory(objectMapper()).create();
+    }
+
+    static Scheduler newScheduler(String pattern, String timeZone)
+    {
+        return new DailySchedulerFactory().newScheduler(newConfig().set("_command", pattern), ZoneId.of(timeZone));
+    }
+
+    private static DateTimeFormatter TIME_FORMAT =
+            DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss Z", ENGLISH);
+
+    static Instant instant(String time)
+    {
+        return Instant.from(TIME_FORMAT.parse(time));
+    }
+
+    @Test
+    public void firstScheduleTimeUtc()
+    {
+        // current time is 09:00:00
+        // schedule is 10:00:00 every day
+        // schedule at today 01:00:00
+        Instant currentTime1 = instant("2016-02-03 09:00:00 +0000");
```

Comment:
If I change this line to `Instant currentTime1 = instant("2016-02-03 01:00:00 +0000");` the test fails.

```
java.lang.AssertionError: 
Expected: is <ScheduleTime{runTime=2016-02-03T10:00:00Z, time=2016-02-03T00:00:00Z}>
     but: was <ScheduleTime{runTime=2016-02-03T01:00:00Z, time=2016-02-02T15:00:00Z}>
Expected :is <ScheduleTime{runTime=2016-02-03T10:00:00Z, time=2016-02-03T00:00:00Z}>
     
Actual   :<ScheduleTime{runTime=2016-02-03T01:00:00Z, time=2016-02-02T15:00:00Z}>
```

Is this expected?